extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -621,4 +624,36 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
         s" [update] privilege on [$db1/$sinkTable1]"))
     }
   }
+
+  test("[KYUUBI #3411] skip checking cache table") {
```

Comment:
cc @yaooqinn . Please have a check on this PR. Some ut in `PrivilegesBuilder` related to create command have updated, do you think we should keep this new added ut as well?